### PR TITLE
Add copy-to-clipboard button for long-lived tokens

### PR DESCRIPTION
**Pull request by lassebm**
## Proposed change

This PR adds a trailing copy-to-clipboard button in the long-lived token text field for easy copying of the somewhat long tokens.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Review by bramkragten — DISMISSED** [review]

**Review by bramkragten — COMMENTED** [review]
Thanks!

File: src/panels/profile/ha-long-lived-access-token-dialog.ts [hunk 1]
```diff
@@ -55,8 +59,15 @@ export class HaLongLivedAccessTokenDialog extends LitElement {
               "ui.panel.profile.long_lived_access_tokens.prompt_copy_token"
             )}
             type="text"
+            iconTrailing
             readOnly
-          ></ha-textfield>
+          >
+            <ha-icon-button
```
Comment (bramkragten) [inline src/panels/profile/ha-long-lived-access-token-dialog.ts:65]: Please add an import for this component.

**Comment by lassebm** [comment]
> Can we add this styling to the `ha-icon-button`:
> 
> ```css
> ha-icon-button {
> 	position: absolute;
> 	top: 10px;
> 	right: 10px;
> 	--mdc-icon-button-size: 36px;
> 	--mdc-icon-size: 20px;
> 	color: var(--secondary-text-color);
> 	inset-inline-start: initial;
> 	inset-inline-end: 10px;
> 	direction: var(--direction);
> }
> ```

The button hover highlight is beneath the text with the styling above. Can you give a hint?

style1

Other dialogs adjust just the icon size, but the right margin seems excessive to me:

```css
ha-textfield > ha-icon-button {
  --mdc-icon-button-size: 24px;
  --mdc-icon-size: 18px;
}
```

style2

**Comment by lassebm** [comment]
Thanks, look great 👍

**Review by bramkragten — APPROVED** [review]

**Pull request merged**